packages/wayland/util/foot/config/foot.ini

```diff
@@ -0,0 +1,197 @@
+# -*- conf -*-
+
+# shell=$SHELL (if set, otherwise user's default shell from /etc/passwd)
+term=xterm
+# login-shell=no
+
+# app-id=foot
+title=foo terminal
+# locked-title=no
+
+font=monospace:size=12
+# font-bold=<bold variant of regular font>
+# font-italic=<italic variant of regular font>
+# font-bold-italic=<bold+italic variant of regular font>
+# line-height=<font metrics>
+# letter-spacing=0
+# horizontal-letter-offset=0
+# vertical-letter-offset=0
+# underline-offset=<font metrics>
+# box-drawings-uses-font-glyphs=no
```

Comment:
do we need to include the outcommented stuff ?

Reply:
depends on your personal flavour but since it's a config file I wouldn't cut it down any further - the space saving... don't mention it